src/pages/blog/[slug].js

```
@@ -27,54 +27,119 @@ const slugToPostContent = ((postContents) => {
+          className="mb-3 d-inline-block btn blog-container__back-btn"
+          href="#"
+          onClick={(event) => navigateToPreviousPage(event)}
+        >
+          <MdKeyboardArrowLeft /> Back
+        </a>
+        <div className="w-100 d-flex align-items-end justify-content-end">
+          {author}, {date}
         </div>
+        {html ? (
+          <div
+            dangerouslySetInnerHTML={html ? { __html: html } : undefined}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)